lib/outcome.rb

```diff
@@ -0,0 +1,27 @@
+class Outcome
+#game logic
+attr_reader :computer_choice, :weapon
+def initialize
+ @computer_choice = ["Rock", "Paper", "Scissors"].sample
+ #@computer_choice = nil
+ #$weapon = $weapon
+end
+
+#attr_reader :computer_choice
+def results(test)
```

Comment:
Hehe, you've taken my approach to variable naming here :) 

What could this be renamed to?